superset-frontend/plugins/plugin-chart-table/src/TableChart.tsx

```diff
@@ -200,6 +200,9 @@ function SelectPageSize({
     <span className="dt-select-page-size form-inline">
       {t('page_size.show')}{' '}
       <select
+        aria-label={`${t('page_size.show')} ${current} ${t(
```

Comment:
Could you provide an example of what this label resolves to?

Reply:
Thanks. Added a screenshot in the PR description and using the macro as suggested.